docs/layers/tile-layer.md

```diff
@@ -142,6 +142,12 @@ Hide tiles when under-zoomed.
 
 - Default: 0
 
+##### `extent` (Array, optional)
+
+If provided, the layer will load and render the tiles in this box when underzoomed.  The box is of the form [left, top, right, bottom]
```

Comment:
This does not apply to geospatial views - I'd suggest `[minX, minY, maxX, maxY]`